dev/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM php:8.1-fpm
+
+RUN curl -sL https://deb.nodesource.com/setup_18.x | bash -
+
+RUN apt-get update && apt-get upgrade -y && apt-get install -y git libicu-dev nodejs \
+    nginx libpng-dev libzip-dev libwebp-dev unzip zip libxml2-dev libjpeg62-turbo-dev zlib1g-dev \
+    libfreetype6 libfreetype6-dev
+
+RUN docker-php-ext-install opcache \
+    && docker-php-ext-configure gd --with-webp=/usr/include/webp --with-jpeg=/usr/include --with-freetype=/usr/include/freetype2/ \
```

Comment:
Have you found something that needs the GD extension?

Reply:
I tested it now and it's really not necessary, it was removed